tests/test_plugins_general_locale.py

```diff
@@ -0,0 +1,74 @@
+import textwrap
+from io import BytesIO
+
+from dissect.target.helpers.regutil import VirtualKey, VirtualValue
+from dissect.target.plugins.os.unix.locale import LocalePlugin as UnixLocalePlugin
+from dissect.target.plugins.os.windows.locale import LocalePlugin as WindowsLocalePlugin
+
+from ._utils import absolute_path
+
+
+def test_locale_plugin_windows(target_win_users, hive_hku, hive_hklm):
+    """
+    Hive structure originates from a Windows Server 2019 installation.
+    """
```

Comment:
```suggestion
    # Hive structure originates from a Windows Server 2019 installation.
```
This is more a code comment than a function docstring.